public/app/plugins/panel/heatmap/suggestions.ts

```diff
@@ -18,7 +18,22 @@ export class HeatmapSuggestionsSupplier {
       return;
     }
 
-    const info = prepareHeatmapData(builder.data, defaultPanelOptions, config.theme2);
+    //avoid type assertion
+    const panelOptions: PanelOptions = {
```

Comment:
This is done in two places now, maybe better create a `types.ts` file and define this there? You could also re-export all the types from `./panelcfg.gen` there to have a single source of truth for the types.

Reply:
Agreed. Set a types.ts single source of truth for the heatmap panel